llvm/test/CodeGen/AArch64/GlobalISel/prelegalizercombiner-shuffle-vector.mir

```diff
@@ -12,10 +12,11 @@ body:             |
 
     ; CHECK-LABEL: name: shuffle_vector_to_concat_vector_0123
     ; CHECK: liveins: $d0, $d1
-    ; CHECK: [[COPY:%[0-9]+]]:_(<2 x s32>) = COPY $d0
```

Comment:
Could you please pre-commit a patch which just updates the checks in this file (no review needed) and then rebase the current patch on top of that?